js/ui/text_box/ui.text_editor.base.js

```diff
@@ -746,6 +750,7 @@ const TextEditorBase = Editor.inherit({
 
         switch(name) {
             case 'valueChangeEvent':
+            case 'customItemCreateEvent':
```

Comment:
there is now such option on text editor => we should not handle its change here

Reply:
https://github.com/DevExpress/DevExtreme/pull/22772/commits/4c59742555c2e9e4b45320926142f9f53fee8cd8